Annex60/Fluid/Interfaces/ConservationEquation.mo

```diff
@@ -60,7 +60,9 @@ model ConservationEquation "Lumped volume with mass and energy balance"
   Modelica.Blocks.Interfaces.RealInput mWat_flow(unit="kg/s")
     "Moisture mass flow rate added to the medium"
     annotation (Placement(transformation(extent={{-140,0},{-100,40}})));
-
+  Modelica.Blocks.Interfaces.RealInput[Medium.nC] C_flow(unit="kg/s")
```

Comment:
@Mathadon : I will remove the unit declaration as for PPM, the unit would be wrong.